k8s/charts/openebs/templates/deployment-local-provisioner.yaml

```diff
@@ -67,7 +67,7 @@ spec:
           exec:
             command:
             - pgrep
-            - ".*localpv"
+            - ".*provisioner-loc"
```

Comment:
I am still not sure why existing option does not work

Reply:
Ok! Figured out the problem. The `localpv-provisioner` I built for ARM had the different base image(OS). The one I was using was ubuntu and existing it was alpine. Since alpine is bundled w/ `busybox`. BusyBox combines tiny versions of many common UNIX utilities into a single small executable. Thus the version of `pgrep` in both mismatched.

**Ubuntu**
```bash
$ pgrep --version
pgrep from procps-ng 3.3.12
```

**Alpine**
```bash
$ pgrep --version
pgrep: unrecognized option: version
BusyBox v1.26.2 (2018-05-30 13:51:20 GMT) multi-call binary.
```

One way to have a generic solution is instead of changing the match pattern, we can add a flag, `-f`, e.g. 

```yaml
livenessProbe:
          exec:
            command:
            - pgrep -f
            - ".*localpv"
```
**man page of pgrep -f**
```bash
PGREP(1) 
SYNOPSIS
       pgrep [options] pattern
     -f, --full
              The pattern is normally only matched against the process name.  When -f is set, the full command line is used.
```

Ref: [BusyBox](https://busybox.net/downloads/BusyBox.html) [Stack Exchange](https://unix.stackexchange.com/a/347202)

Reply:
Full name sounds like a good option @harshvkarn - can you update the PR with the change. Also provide the results of running with the modified changes on both arm and amd clusters.